### PR TITLE
feat(icons): add workflow icon

### PR DESCRIPTION
**Pull request by frogzsj**
image

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @frogzsj has signed the CLA. Thanks for the pull request!

File: src/icons/general/IconWorkflow.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconWorkflow = ({ className = '', color = '#999999', height = 32, title, width = 32 }: Props) => (
```
Comment (booc0mtaco) [inline src/icons/general/IconWorkflow.js]: could we swap the default here with `$nines` (or the closest BDL color `$bdl-neutral-03`)?

File: src/icons/general/IconWorkflow.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconWorkflow = ({ className = '', color = seesee, height = 32, title, width = 32 }: Props) => (
```
Comment (priyajeet) [inline src/icons/general/IconWorkflow.js]: should the color be one of a bdl colors by default?

File: src/icons/general/IconWorkflow.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    <AccessibleSVG
+        className={`icon-watermark ${className}`}
```
Comment (priyajeet) [inline src/icons/general/IconWorkflow.js]: bdl-IconWorkflow

File: src/icons/general/IconWorkflow.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        height={height}
+        title={title}
+        viewBox="0 0 32 32"
+        width={width}
+    >
+        <path
+            d="M6,2H26a4.01176,4.01176,0,0,1,4,4V26a4.01176,4.01176,0,0,1-4,4H6a4.01176,4.01176,0,0,1-4-4V6A4.01176,4.01176,0,0,1,6,2Zm9,14h4.5a3.5,3.5,0,0,0,0-7H14a.94477.94477,0,0,0-1,1,.94477.94477,0,0,0,1,1h5.5a1.5,1.5,0,0,1,0,3H13a4,4,0,0,0,0,8h3a1,1,0,0,0,0-2H13a2,2,0,0,1,0-4Zm-5-5a.94477.94477,0,0,0,1-1,.94477.94477,0,0,0-1-1,.94477.94477,0,0,0-1,1A.94477.94477,0,0,0,10,11ZM21,23a2,2,0,1,0-2-2A2.00588,2.00588,0,0,0,21,23Z"
+            fill={color}
```
Comment (priyajeet) [inline src/icons/general/IconWorkflow.js]: missing className="fill-color"

File: src/icons/general/IconWorkflow.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        />
+    </AccessibleSVG>
+);
+
+export default IconWorkflow;
```
Comment (priyajeet) [inline src/icons/general/IconWorkflow.js]: Missing test file

File: src/icons/general/IconWorkflow.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    color?: string,
+    height?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconWorkflow = ({ className = '', color = bdlNeutral03, height = 32, title, width = 32 }: Props) => (
+    <AccessibleSVG
+        className={`bdl-icon-workflow ${className}`}
```
Comment (priyajeet) [inline src/icons/general/IconWorkflow.js]: bdl-IconWorkflow

File: src/icons/general/__tests__/IconWorkflow-test.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import IconWorkflow from '../IconWorkflow';
+
+describe('icons/general/IconWorkflow', () => {
+    test('should correctly render default icon', () => {
```
Comment (priyajeet) [inline src/icons/general/__tests__/IconWorkflow-test.js]: Use snapshot testing for icons

**Comment by frogzsj** [comment]
@priyajeet addressed your comments 👍

**Pull request merged**